### DISCUSSION TITLE
Visitor profiles

### DISCUSSION THREAD

**Discussion by grandeljay** [plausible/analytics#892] [Anything Else]
Something I liked about matomo is that even though the visitor remained anonymous, I was still able to select and see certain profiles of users.

What kind of Hardware, Software, Resolution, etc are they using? I liked seeing a grouped dashboard of a visitor, even if I didn't know who he/she was.

I think this is a useful insight and would really appreciate seeing this in plausible!

**Comment by metmarkosaric** [comment]
thanks @grandeljay! you can already see this in the Devices report: devices people use, browsers and OS too. all in aggregate and private way. we don't plan to show individuals and their individual stats if that's what you meant as we're privacy first.

**Reply by grandeljay** [reply]
From my point of view, seeing individuals would still be just as private. There still wouldn't be any way to tie the information to a said person, since IP and whatnot is still anonymised.

Am I missing something here?

**Reply by metmarkosaric** [reply]
the whole premise of Plausible is to be privacy-first and to not track unnecessary information just for the sake of tracking. the useful info you can already see in aggregate numbers (how many people use specific device/browser/os) and there's no extra insights you can get by tracking individuals.
